### DISCUSSION TITLE
[Feature] Mark an actress or studio as curated

### DISCUSSION THREAD

**Discussion by miltuss** [stashapp/stash#5295] [Q&A] [answered]
Hello, it is currently possible to mark scenes or images as organized, but it is not possible to mark performers or studios as organized.  It would be interesting to do this to know if all the scenes from a studio or a performer have been downloaded for example.

**Comment by feederbox826** [comment]
A simple workaround could just be to add a `Curated` Tag to the performer and studio, and you can add your markers/ indicators off of that

**Comment by randemgame** [comment, marked as answer]
see https://github.com/stashapp/stash/issues/4970
